packages/shared/models/task.js

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const taskSchema = new mongoose.Schema({
+  id: String,
+  title: String,
+  description: String,
+  captionURL: String,
+  attachments: [
+    {
+      id: String,
+      URL: String,
+      fileName: String,
+      createdAt: String
+    }
+  ],
+
+  comments: [
+    {
+      id: String,
+      createdAt: String,
+      content: String,
+      user: {
+        id: String,
+        avatarURL: String,
```

Comment:
why do you want to store image URL? We will store image as JSON. We should probably remove this field because Mongo has limitation for document 16 MB.

Reply:
change to avatarId: Number